evals/registry/modelgraded/sql.yaml

```diff
@@ -1,24 +1,25 @@
-prompt: |-
+sql:
+  prompt: |-
   You are comparing a submitted answer to an expert answer on a given SQL coding question. Here is the data:
```

Comment:
You need to add two spaces here. Indentation is off here.

Reply:
PR #1052 corrects this.